src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -320,15 +310,7 @@ ManifestCache::load (
                 continue;
             }
 
-            if (unl.listed(mo->masterKey))
-            {
-                applyManifest (std::move(*mo), unl);
-            }
-            else
-            {
-                JLOG(j_.info())
-                   << "Manifest in db is no longer trusted";
-            }
+            applyTrustedManifest (std::move(*mo));
```

Comment:
Yeah, you could run while trusting a validator in your config (`[validators]`) and receive its manifest from the network. Then you could remove it from your config and restart.
There's minimal downside to having an untrusted manifest in the cache. You'd do the signature verification work and forward it to peers at start up. You'd finally drop it the next time you shutdown.